programs/GrabberControls2Gui/GrabberControls2GuiGUI.py

```diff
@@ -102,6 +102,18 @@ def setupUI(self):
         self.fpsSpinBox.setMinimum(0)
         self.fpsSpinBox.setMaximum(30)
 
+        # Disable widgets if feature not supported
+        self.zoomSlider.setEnabled(False) if not self.controller.has_zoom() else None  # Look me mom I'm a hacker!
```

Comment:
Tip for pro hackers 😄:

```python
not self.controller.has_zoom() and self.zoomSlider.setEnabled(False)
```

Although this feels far less obscure:

```python
self.zoomSlider.setEnabled(self.controller.has_zoom())
```

Or just group in pairs like an old-sk00ler, provided `True` is the default value (it seems so):

```python
if not self.controller.has_zoom():
    self.zoomSlider.setEnabled(False)
    self.zoomSpinBox.setEnabled(False)
```